doc/man/scons.xml

```diff
@@ -511,11 +515,11 @@ and many of those supported by <application>cons</application>.
     <option>--remove</option>
   </term>
   <listitem>
-<para>Clean up by removing all target files for which a construction
-command is specified.
-Also remove any files or directories associated to the construction command
-using the &Clean; function.
-Will not remove any targets specified by the &NoClean; function.</para>
+<para>Clean up by removing the specified targets and their
```

Comment:
Maybe add a bit about if no target is specified it will clean all targets which would be built by default?

Reply:
I don't have a problem with that, I had written something like that, then decided to let *specified targets* be the coverall for that, since it's already defined what specifying/not specifying targets means and it didn't feel fully necessary to repeat.  If you don't mind "repeating" here, it would make it more explicit to add.